app/views/passthrough/error.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Permission Denied</h1>
+<p>Please check that your Signon organisation ID is correct</p>
```

Comment:
Perhaps we could change this so that it matches the other message, something like:

>You aren't permitted to access this application. If you feel you've reached this in error, contact your SPOC.